examples/fft_conv.cu

```diff
@@ -129,18 +138,30 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
     
   }
 
+#if 0
   cudaEventRecord(stop, stream);
```

Comment:
Eventually we should mask these events behind the executor as well so the timing is the same regardless of the executor.

Reply:
Yes this makes it look like the code is very different for both executors but that timing is the sole reason especially if finalize is moved to the dtor